### DISCUSSION TITLE
Unable to reorder views

### DISCUSSION THREAD

**Discussion by armandgrillet** [community/community#5771] [Projects and Issues] [answered]
The views in a new org cannot be reordered, it would be great if I could simply drag and drop them.

I'm talking about those for example ("Board" and "Epics"):
<img width="250" alt="PS 2021-09-23 à 16 17 54" src="https://user-images.githubusercontent.com/2117580/134524224-3b49f1ea-8bd7-441e-b4cf-a757c81e62b2.png">

**Comment by hashboard-charlie** [comment, marked as answer]
Duplicate of #4803
